web/src/js/Menu.jsx

```diff
@@ -30,7 +38,7 @@ export default class Menu extends Component {
         <div className="collapse navbar-collapse" id="navbarNavAltMarkup">
           <div className="navbar-nav">
 {this.menuItems.map((menuItem, index) =>
-            <a key={index} className={"nav-item nav-link "+(window.location.pathname == menuItem.link ? "active" : "")} href={menuItem.link}>{menuItem.name}</a>
+            <a key={index} className={`nav-item nav-link ${window.location.pathname == menuItem.link ? "active" : ""} ${menuItem.requireLogin && !isLoggedIn() ? 'd-none' : ''}`} href={menuItem.link}>{menuItem.name}</a>
```

Comment:
Would appreciate if you could do this with a `.filter()` call before the `.map()`

Reply:
Done